src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,66 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, S> {
+    private T integer;
+    private S string;
+
+    private Pair(T integer, S string) {
+        this.integer = integer;
+        this.string = string;
+    }
+
+    public static <T, S> Pair<T, S> of(T integer, S string) {
+        return new Pair<>(integer, string);
+    }
+
+    public T getFirst() {
+        return integer;
+    }
+
+    public S getSecond() {
+        return string;
+    }
+
+    public T getInteger() {
+        return integer;
+    }
+
+    public void setInteger(T integer) {
+        this.integer = integer;
+    }
+
+    public S getString() {
+        return string;
+    }
+
+    public void setString(S string) {
+        this.string = string;
+    }
```

Comment:
let's remove duplicate getters and rename setters

getFirst() and getSecond() are supposed to be getters)